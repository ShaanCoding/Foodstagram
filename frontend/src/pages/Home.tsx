import React from 'react'
import styles from '../styles/Home.module.css'

const Home = () => {
	return <div className={styles.container}>Home Updated</div>
}

export default Home
