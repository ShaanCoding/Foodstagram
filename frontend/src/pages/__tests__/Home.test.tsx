import { QueryClient, QueryClientProvider } from 'react-query'
import { BrowserRouter, Router } from 'react-router-dom'
import renderer from 'react-test-renderer'
import Home from '../Home'

const queryClient = new QueryClient()

it('The Home page renders correctly', () => {
	const tree = renderer
		.create(
			<QueryClientProvider client={queryClient}>
				<BrowserRouter>
					<Home />
				</BrowserRouter>
			</QueryClientProvider>
		)
		.toJSON()
	expect(tree).toMatchSnapshot()
})
