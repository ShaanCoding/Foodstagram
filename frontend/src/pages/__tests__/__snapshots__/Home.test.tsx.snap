// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Home page renders correctly 1`] = `
<div>
  Home Updated
</div>
`;
